### ISSUE TITLE
Feedback: Modelling interventions

### ISSUE THREAD

**Issue by CarmenTamayo** [epiverse-trace/tutorials#73]
1. •	Overall, I feel like the tone seems more like a case study than a tutorial/teaching material. We have the plan to include late stage tasks in {episoap} as well as part of the pipelines, and this is the kind of format that such pipeline would take, as opposed to interactive, problem-solving materials. I believe the content could be instead posed as questions that students can answer using {epidemics}, e.g., instead of saying that closures will result in a reduction of transmission and then showing the output, a question for the reader could be included about how they think that a reduction in contacts would affect the transmission, and how this could be visualised using {epidemics}, maybe including hints so that they can do this without constantly moving back and forth between this episode and the one about simulating transmission. Maybe we could have a discussion about this, I was envisioning the tutorials as a series of practical exercises, with a theoretical introduction, and solutions provided at the end of each exercise, so that readers can learn as they complete the tutorial. Right now this seems more like a vignette with low levels of interaction from the user.
2. •	Minor aesthetic point: throughout the episodes, the first table that contains the objectives and questions- the columns are not aligned, and it doesn't look good that the objective lines are piled up and then the question lines take up so much horizontal space. It'd be good to distribute the space evenly between the columns.

3. •	"Mathematical models can be used to generate predictions for the implementation of non-pharmaceutical and pharmaceutical interventions at different stages of an outbreak" <- this is okay but a bit vague and potentially slightly confusing for readers that are new to the topic, I would clarify that the predictions (or projections- I think it's important to check the vocabulary to use as well) can be used to make decisions about which measures to implement to contain the spread of the outbreak. If you don’t think it's the right place to elaborate on this, I would at least include a couple of references with examples where the outputs of models were used to guide measure implementation. It's likely that a big portion of students will be those that at some point and to some degree will be making these decisions themselves, so it's important to make the utility of this episode very apparent to readers.
4. •	"Non-pharmaceutical interventions (NPIs) are measures put in place to reduce transmission that do not include taking medicine or vaccines" <- This could sound more scientific, e.g., "..measures to reduce tx that do not include the administration of drugs or vaccinations", and also should be included in the glossary, perhaps with a link to further resources, such as:  https://www.gov.uk/government/publications/technical-report-on-the-covid-19-pandemic-in-the-uk/chapter-8-non-pharmaceutical-interventions
5. •	"In mathematical modelling, we must make assumptions about how NPIs will affect transmission. This may include adding additional disease states or reducing the value of relevant parameters." <- I think that sentence located in the intro is a bit vague and doesn't add a lot of information for the reader, I'd suggest removing it and instead writing a paragraph about why we would want to model interventions, why that is useful, and provide some examples of applications that might be relevant to the different students using these tutorials.
6. •	Before the section about COVID closures, or in the overall intro to the episode, I'd add some context about how specific examples will be used to illustrate the NPIs, especially since this is not done in the section of pharmaceutical interventions, where the content goes straight into the model characteristics and the R code- where does the content/data for this section come from? Is it also COVID? This needs clarification.
7. •	Again, I'm not 100% convinced that students would need to complete the "Simulating tx" episode to be able to follow along the other episodes. I think a brief intro to the concept and mention of the necessary functions on {epidemics} should be enough? Rather than referencing the entire episode, maybe I would reference the specific relevant parts, for instance, the contact matrix section, which would help understand the "LOAD CONTACT AND POPULATION DATA" part of the present episode.
8. •	I'm not entirely sure of the utility of the "EFFECT OF INTERVENTIONS ON CONTACTS" box, if {epidemics} (to my understanding) doesn't require these updated proportions as an input, i.e., the package takes in the desired % and does the math directly. I think it might be confusing to users, who may think that what they have to input is the updated proportions after manually doing the calculation themselves… I understand that the intention is to show how this is actually calculated, but I think it's too detailed for someone who is learning how to model interventions and also learning how to use the R package… maybe it could be shown later on in the episode to demonstrate what is actually happening to the matrix, but only after readers have understood the whole process and gone through it at least once.
9. • Is the plot an outcome of "output_school"? Where does it come from? How can a reader replicate it? This is the kind of info that is useful in a tutorial vs a case study.
10. •	The "effect of mask wearing" section should include a reference to {epidemics} vignettes, or, for example, a text box with a reminder on how to check for the arguments of the functions, so that they can understand the meaning of elements and acceptable options for arguments such as "type".
11. • "In this case, it is useful to add another disease state…" <- in what case? Another on top of what? (I understand it means building on the SEIR model for NPI, but it's lacking context and will probably confuse readers)- it also doesn't say why. Then the tutorial moves on to the equations of the model, without mentioning where they come from/how are they different from the previous model/why they are included here but not for the previous model/whether this means the model in {epidemics} has now changed, and also without a legend to explain the equations (as I mentioned on the previous episode, I think this is probably too advanced for beginner users to understand, and that, if context is not added, they should be removed as they don't add much to the practical exercise). 
11.1 - "equations are described in the tutorial Simulating transmission" <- which section? Also, why not include a hyperlink when other episodes' sections are mentioned to facilitate the process?
11.2. - How is the vaccination compartment included in "model_default_cpp"? The vaccination is included as an object, but so were the NPIs, this needs clarification as readers won't know how to interpret this.
12. •	Overall I think the pharmaceutical interventions section is not as developed as the NPI one, there is not a lot of content and little explanation about the provided R code. I don't think the content is easy to follow or understand, especially for someone who is new to modelling and new to Epiverse packages.
13. •	"To explore the effect of vaccination we need to create a vaccination object" <- this needs context, we need this for the {epidemics} R package, can other pharmaceutical interventions be included as well, e.g., treatment administration? If so, how can a user do this? If not included in the practical, maybe recommend as further reading through vignettes.
14. •"Here we see that the total number of infectious individuals when vaccination is in place is much lower compared to school closures and mask wearing interventions" how can a student see this? What is the code that they can use to replicate the plot? Where does the info about mask wearing come from???
15. •	The "Summary" section appears here and on the "Simulating transmission" episode, but not on the rest.
15.1. - "Modelling interventions requires assumptions of how interventions affect model parameters such as contact matrices or parameter values." <- this is the only summary and it doesn't accurately reflect what students have actually learned from this episode. The summary should be an overview of the contents of the episode.
16. •	"Next we want quantify the effect of an interventions. In the next tutorial, we will learn how to compare intervention scenarios against each other." <- I'd remove this, it's not a summary of the current episode, but also because more episodes might be added later on, meaning that these 2 might not be adjacent anymore. Also a student also has the option of doing these tutorials as standalone practicals. Instead, I'd write something like "If you are interested in learning how to compare interventions, please complete the episode X (with a hyperlink)".
17. •	Lastly, the key points section only includes one key point, "Different types of intervention can be implemented using mathematical modelling", which actually sounds more like a summary than a key point? I believe the "key points" sections throughout the episodes should either be improved with stronger points about useful info and take-away messages that students have learned/key applications that might be relevant to our target audience, or removed entirely, as right now I feel they're downplaying the importance of the tutorials.

**Comment by amanda-minter** [comment]
> 5. •	"In mathematical modelling, we must make assumptions about how NPIs will affect transmission. This may include adding additional disease states or reducing the value of relevant parameters." <- I think that sentence located in the intro is a bit vague and doesn't add a lot of information for the reader, I'd suggest removing it and instead writing a paragraph about why we would want to model interventions, why that is useful, and provide some examples of applications that might be relevant to the different students using these tutorials.

Agree this can be added to the introduction, I will add as a separate issue
